portacle.el

```diff
@@ -62,3 +63,50 @@
 (require 'portacle-ag)
 (require 'portacle-help)
 (require 'portacle-user)
+
+;;; Setup SLIME or SLY to start at
+;;;
+(defun portacle-let-user-choose-ide (implementation)
+  (let* ((ide-question
+          (concat
+           "Portacle comes with SLY and SLIME, two great common Lisp IDE's."
+           "\n"
+           "Which one would you like to start now? "))
+         (choice
+          (intern
+           (ido-completing-read ide-question'("slime" "sly") nil t nil nil)))
+         (save-question
+          (concat
+           (format
+            "Great! Use `%s' from now on? " choice)
+           "You can customize `portacle-ide' later if you change your mind."))
+         (savep (y-or-n-p save-question)))
+    (when savep
+      (customize-save-variable 'portacle-ide choice))
+    (funcall choice implementation)))
```

Comment:
I would really, really, rather that the choice of IDE is part of `portacle-configure` and it default to Slime on first start. I don't want to have any necessary input prompts on first start.

Reply:
OK, I see your point. What if I get rid of the second prompt and make it a `yes` by default? In that case we're talking about a one-time-ever keypress of "enter", the most prominent key. 

Because, you know, if I spent the effort to get SLY into Portacle, I think it's fair to afford it a decent chance of discoverability, if undestandably not make it the default choice.